app/controllers/itt_providers_controller.rb

```diff
@@ -15,9 +15,7 @@ def update
     if @itt_provider_form.save
       next_question
     else
-      if FeatureFlag.active?(:use_dqt_api_itt_providers)
-        set_itt_provider_options
-      end
+      set_itt_provider_options
```

Comment:
Ah, this is a different feature flag; we haven't switched this on yet. Please leave it in :+1: